WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -133,6 +134,17 @@ private void StepConnectionConfirmationPhase()
 					var removedAliceCount = round.Alices.RemoveAll(x => alicesDidntConfirm.Contains(x));
 					round.LogInfo($"{removedAliceCount} alices removed because they didn't confirm.");
 
+					if (round.InputCount >= Config.MinInputCountByRound)
```

Comment:
It would be great to have the following comment here:

> In order to prevent round denial of service, the spend status of all registered inputs should be checked right before switching to the connection confirmation phase.
#6074

But I know you guys don't like to comment too excessively. So feel free to ignore the comment.

Reply:
Comments that explain what a code are harmful because they add more text without adding meaning, but comments that explain why the code is there when it's not obvious are valuable.

The rationale for this behavior is that once an input is confirmed and non-zero credentials are issued, it must be included and must provide a signature for a valid transaction to be produced, therefore this is the last possible opportunity to remove any problematic inputs. This might be obvious to someone who is very familiar with the concepts, but for someone trying to understand the code the first time I think such a comment is a good idea.

Anyway, :+1:

Reply:
Comment added. But it is rather obvious that nobody can participate with an already spent coin.